GoogleForCreators/web-stories-wp
# Enforce Canvas to be LTR to support Moveable

## Summary



## Relevant Technical Choices


## To-do



## User-facing changes



## Testing Instructions
- Turn on RTL Tester plugin or change your locale to RTL language.
- Add some elements, move them around and resize
- Verify that the elements move as expected
- Test that left and right arrow keys move the elements as expected
- Test that the text written in the edit mode is still working RTL. (Cursor is in the "beginning" of the text for LTR people)


---



Fixes #508
